app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="recipe-container">
+  <div class="recipe-details">
+    <h1><%= @recipe.name %></h1>
+    <p><strong>Preparation Time:</strong> <%= @recipe.preparation_time %> minutes</p>
+    <p><strong>Cooking Time:</strong> <%= @recipe.cooking_time %> minutes</p>
+    <p><strong>Description:</strong> <%= @recipe.description %></p>
+    <p><strong>Public:</strong> <%= @recipe.public ? 'Yes' : 'No' %></p>
+  </div>
+    <%= link_to 'Back to Recipes', recipes_path, class: 'action-button' %>
+</div>
+
+<div class="public">
+  <% if @recipe.user_id == current_user.id %>
+    <%= form_for @recipe, url: recipe_path(@recipe), method: :patch, class: "form-public" do |f| %>
+      <label class="label-public">
+        <%= f.check_box :public, class: "public-checkbox", checked: @recipe.public, onclick: "this.form.submit();" %>
+        <span class="slider round"></span>
+        <p class="recipe-label-text">Public</p>
+      </label>
+    <% end %>
+  <% end %>
+</div>
+
+<div class="recipe-buttons">
+  <% if user_signed_in? %>
+  <%= link_to "Generate Shopping List", shopping_lists_path, class: "action-button", method: :get %>
+<% end %>
+
+  <% if @recipe.user == @current_user %>
+    <%= button_to "Add Ingredients", new_recipe_recipe_food_path(@recipe.id), method: :get, class: "action-button" %>
+  <% end %>
+</div>
+
+<hr>
+<table class="recipe-table">
+  <thead>
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Value</th>
+      <% if @recipe.user == current_user %>
+        <th>Action</th>
+      <% end %>
+    </tr>
+  </thead>
+  <tbody>
+    <%= render partial: "recipe_foods/index", locals: { recipe_foods: @recipe.recipe_foods } %>
+  </tbody>
+</table>
```

Comment:
- [ Optional ] Good job implementing the changes from the previous reviewer, I would like to mention a few things that could improve your design. Currently, your design varies with the project wireframe in the following ways. I highly recommend you fix these inorder to improve your project.

    
   - [ ] The `Add ingredient` and `Generate shopping list` buttons are not aligned the same way they are aligned on the wireframe design.
   - [ ] The ingredient list is not styled the same way it has been styled in the project wireframe. If you look at the project wireframe it is zebra stripped, please see the screenshot below for more details on this.

## Project wireframe
![](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/recipe_food.png?token=GHSAT0AAAAAABTYBUDYDATDKIWVLNMEQB5IZFAFADQ)

## Your design
![Screenshot from 2023-07-01 15-12-21](https://github.com/ngeelani48/Recipe-app/assets/16392046/03a8dfad-2956-407a-8962-3c5c91d187c2)

Reply:
@Felix45, Thanks for the suggestion we will work on it. 👍